tests/integration/components/public/event-map-test.js

```diff
@@ -0,0 +1,19 @@
+import { test } from 'ember-qunit';
+import moduleForComponent from 'open-event-frontend/tests/helpers/component-helper';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent('public/event-map', 'Integration | Component | public/event map', {
```

Comment:
Please see how the test is defined in the other component test files. Use the custom `moduleForComponent` helper.